cluster/examples/coreos/ceph-reboot-check-script.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ceph-reboot-check-script
+  namespace: rook
+data:
+  status-check.sh: |-
+    #!/bin/bash
+    # Wait 30 second to be sure ceph status has latest info
+    echo "ROOK-UPDATE-SCRIPT:: Running the rook initiation script..."
+    /usr/local/bin/toolbox.sh &
+
+    echo "ROOK-UPDATE-SCRIPT:: Sleeping for 30 seconds to make sure Ceph has the latest cluster status (see ROOK_MON_HEALTHCHECK_INTERVAL in the rook-operator manifest)"
```

Comment:
You don't need to wait 30 seconds here. The 30 seconds `ROOK_MON_HEALTHCHECK_INTERVAL` is for the mon "internal" healthcheck in the Rook operator. Ceph will "always" expose the cluster status.
You may only want to wait 3 seconds or if the `/etc/ceph/ceph.conf` to exist till the toolbox.sh script first ran.